charts/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,308 @@
+    to specify the `kind` (Issuer or ClusterIssuer) and the `name`. Noted that this method also requires the installation of cert-manager.
+  - The last way is to manually modify the secret where the TLS certificate is stored. You can either do this before installation
+    or after.
+    - To do this before installation, you don't have to install cert-manager. Please set `admissionWebhooks.certManager.enabled` to `false`.
+      - Create the namespace for the OTEL Operator and the secret
+        ```console
+        $ kubectl create namespace opentelemetry-operator-system
+        ```
+      - Config the TLS certificate
+        ```console
+        $ kubectl apply -f - <<EOF
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: opentelemetry-operator-controller-manager-service-cert
+            namespace: opentelemetry-operator-system
+          type: kubernetes.io/tls
+          data:
+            tls.crt: |
+                # your signed cert
+            tls.key: |
+                # your private key
+          EOF
+        ```
+        You can also do this using `kubectl create` command.
+        ```console
+        $ kubectl create secret tls opentelemetry-operator-controller-manager-service-cert \
+            --cert=path/to/cert/file \
+            --key=path/to/key/file \
+            -n opentelemetry-operator-system
+        ```
+    - To do this after installation, you will have to install cert-manager first. Once the Operator is ready, you can override
+      the secret by putting your certificate there. What you need to do is the same as the second step above.
+
+## Add Repository
+
+```console
+$ helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+$ helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+The following command installs the chart with the release name `my-opentelemetry-operator` in the namespace `opentelemetry-operator-system`.
+You don't need to provide `--namespace` option here.
+```console
+$ helm install \
+  my-opentelemetry-operator open-telemetry/opentelemetry-operator
+```
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Uninstall Chart
+
+The following command uninstalls the chart whose release name is my-opentelemetry-operator.
+
+```console
+$ helm uninstall my-opentelemetry-operator
+```
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+This will remove all the Kubernetes components associated with the chart and deletes the release.
+
+The OpenTelemetry Collector CRD created by this chart won't be removed by default and should be manually deleted:
+
+```console
+$ kubectl delete crd opentelemetrycollectors.opentelemetry.io
+```
+
+The namespace created should also be removed manually:
+
+```console
+$ kubectl delete ns opentelemetry-operator-system
+```
```

Comment:
Since you defined namespace in the yaml instead of `install ... --create-namespace`, this step is not needed anymore, namespace will be removed with `helm uninstall`

Reply:
Yes, I overlooked that. I will remove this step. Thank you!